Lib/fontbakery/specifications/name.py

```diff
@@ -372,3 +372,28 @@ def com_google_fonts_check_163(ttFont):
                                 unidecode(stylename_str))
   if not failed:
     yield PASS, "All name entries are good."
+
+
+@check(
+  id='com.google.fonts/check/postscript_name_cff_vs_name',
+  conditions=['is_cff']
+)
```

Comment:
I thought that this should be a general check for everyone, per discussion with Miguel, but now I'm not as certain. I'll check with Miguel and then either write a rationale or move this into an Adobe-specific check depending on what we conclude.

Reply:
Just realized that I committed an incomplete rationale text. I'll update it after talking with Miguel.

Reply:
@miguelsousa confirmed that this check should be used by everyone, it's not Google or Adobe specific. I added a rationale based on our discussion.

Reply:
Just to be clear, I put this in `name.py` so that everyone would get it.

I'm planning to create an `adobe-fonts` profile for fonts that will be released by Adobe Fonts (formerly Typekit), and an `adobe-originals` profile for fonts created by the Adobe Type Team.